docs/standard/frameworks.md

```diff
@@ -14,139 +14,150 @@ ms.assetid: 6ef56a2e-593d-497b-925a-1e25bb6df2e6
+
+```csharp
+public class MyClass
+{
+    static void Main()
+    {
+#if NET40
+        Console.WriteLine("Target framework: .NET Framework 4.0");
+#elif NET45  
+        Console.WriteLine("Target framework: .NET Framework 4.5");
+#else
+        Console.WriteLine("Target framework: .NET Standard 1.4");
+#endif
+    }
+}
+```
+
+The build system is aware of preprocessor symbols representing the target frameworks shown in the [Supported target framework versions](#supported-target-framework-versions) table. When using a symbol that represents a .NET Standard or .NET Core TFM, replace the dot in the version with an underscore (for example, the symbol for `netstandard1.4` is `NETSTANDARD1_4`).
+
+The complete list of preprocessor symbols for .NET Core target frameworks is:
+
+[!INCLUDE [Preprocessor symbols](~/includes/preprocessor-symbols.md)]
+
+It's less likely that developers cross-target .NET Core app target frameworks compared to cross-targeting for libraries, but you can cross-target app target frameworks when needed:
+
+```xml
+<TargetFrameworks>netcoreapp1.0;netcoreapp1.1</TargetFrameworks>
+```
+
+```csharp
+public static void Main(string[] args)
+{
+#if NETCOREAPP1_1
+    Console.WriteLine("Hello World from netcoreapp1.1!");
+#else
+    Console.WriteLine("Hello World from netcoreapp1.0!");
+#endif
+}
+```
+
+## Deprecated target frameworks
+
+The following target frameworks are deprecated. Packages targeting these target frameworks should migrate to the indicated replacements.
+
+| Deprecated TFM                                                                             | Replacement |
+| ------------------------------------------------------------------------------------------ | ----------- |
+| aspnet50<br>aspnetcore50<br>dnxcore50<br>dnx<br>dnx45<br>dnx451<br>dnx452                  | netcoreapp  |
+| dotnet<br>dotnet50<br>dotnet51<br>dotnet52<br>dotnet53<br>dotnet54<br>dotnet55<br>dotnet56 | netstandard |
+| netcore50                                                                                  | uap10.0     |
+| win                                                                                        | netcore45   |
+| win8                                                                                       | netcore45   |
+| win81                                                                                      | netcore451  |
+| win10                                                                                      | uap10.0     |
+| winrt                                                                                      | netcore45   |
+
+## Precedence
 
-The [.NET Standard](https://github.com/dotnet/standard) simplifies references between binary-compatible frameworks, allowing a single target framework to reference a combination of others. For more information, see the [.NET Standard](net-standard.md) topic.
+A number of target frameworks are related to and compatible with one another but not necessarily equivalent:
```

Comment:
> we add more explanatory text

Sure ... as soon as I figure out what it means. Remember what I said on my call with u about changing the language too much??!! :grinning: lol